cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Cli/Query.hs

```diff
@@ -55,15 +85,19 @@ hprop_cli_queries = integrationWorkspace "cli-queries" $ \tempAbsBasePath' -> H.
         { cardanoEpochLength = 100
         , cardanoSlotLength = 0.1
         , cardanoNodeEra = cEra
+        , cardanoActiveSlotsCoeff = 0.5
```

Comment:
Interesting. Does it improve stability? Can you add a comment why we need a different than default value here?

Reply:
I am not sure what the effect is, but according to @disassembler:

> epoch length should always be securityparam * 10 / slotCoeff

and because we changed the `epochLength` to 100, `securityParam` is 5, and `slotCoeff` was 0.05, then that had stopped being true.

I noticed something was wrong because, when we try to compute the stability period in the epoch, by using `4 * securityParam / slotCoeff`, I was getting `400`, which is larger than the epoch length (`100`), so that made no sense

Reply:
You can also see [this discussion in slack](https://input-output-rnd.slack.com/archives/CCRB7BU8Y/p1722515058261859)

Reply:
I added a comment here: https://github.com/IntersectMBO/cardano-node/commit/63c852a5823b666d372e585b2c5bb4921bd6b016